mon-pix/app/components/challenge-statement.js

```diff
@@ -68,6 +80,18 @@ export default class ChallengeStatement extends Component {
     });
   }
 
+  _removeMarkDown(text) {
+    let cleanText;
+
+    cleanText = text.replaceAll(/\*\*(.+?)\*\*/g, '$1');
+    cleanText = cleanText.replaceAll(/_(.+?)_/g, '$1');
+    cleanText = cleanText.replaceAll(/~~(.+?)~~/g, '$1');
+    cleanText = cleanText.replaceAll(/`(.+?)`/g, '$1');
+    cleanText = cleanText.replaceAll(/```(.+?)```/g, '$1');
+
+    return cleanText;
+  }
+
```

Comment:
On a essayé coté pix1d notamment avec des anciens navigateurs tel que safari 11. La méthode replaceAll n'est pas supportée. On a trouvé cette autre façon de faire qui devrait être compatible tout navigateur
`
```suggestion
      _removeMarkDown(text) {
    let cleanText = text;
    cleanText = cleanText.split(/[*`_~]/g).join('');
    cleanText = cleanText.split(/-/g).join(' ');
    return cleanText;
    }
  }

```

Reply:
je pensais que le polyfill du build comblerait le manque au build.  

Si dans un texte tu as un asterix qui n'est pas du markdown il disparait aussi non ave ce split ?